packages/compile-solidity/src/polyfill.ts

```diff
@@ -0,0 +1,11 @@
+// must polyfill AbortController to use axios >=0.20.0, <=0.27.2 on node <= v14.x
```

Comment:
Minor nit: I think the filename should reflect the type of polyfill, like `abort-controller-pollyfill.ts` or something similar, instead of `polyfill.ts`

Reply:
I guess we shouldn't worry about this If the intention is to revisit and have a polyfill module as followup work.